plugins/de.cognicrypt.staticanalyzer/src/de/cognicrypt/staticanalyzer/handlers/StartupHandler.java

```diff
@@ -97,16 +118,27 @@ public void resourceChanged(final IResourceChangeEvent event) {
 			}
 			if (!changedJavaElements.isEmpty()) {
 				final AnalysisKickOff ako = new AnalysisKickOff();
-
-				if (ako.setUp(changedJavaElements.get(0))) {
-					if (ako.run()) {
-						Activator.getDefault().logInfo("Analysis has finished.");
-					} else {
-						Activator.getDefault().logInfo("Analysis has aborted.");
-					}
-				} else {
+				boolean stat = ako.setUp(changedJavaElements.get(0));
+				if(stat) {
+					Analysis_Queue.add(ako);
+				}
+				else {
 					Activator.getDefault().logInfo("Analysis has been canceled due to erroneous setup.");
 				}
+				while(Analysis_Queue.size()>0) {
+					if(!analysis_running_flag) {
+						final AnalysisKickOff ak= Analysis_Queue.remove();
+						analysis_running_flag=true;
+						if (ak.run()) {
+							Activator.getDefault().logInfo("Analysis has finished.");
+						} else {
+							Activator.getDefault().logInfo("Analysis has aborted.");
+						}
+						analysis_running_flag=false;
+					}
+					
+				}
+				
 			}
 		}

```

Comment:
Looks good. I've only got some issues with the formatting. 
1. There should be a space on both sides of operators (e.g. greater than, equal sign). 
2. There should be no empty lines (see lines 139 and 141)
3. "} else {" should be on the same line.
4. Organize imports. I doubt the imports in lines 14, 31, and 32 are actually needed.
5. Please also describe the role of the parameters and exceptions in the javadoc.
6. Field names should not begin with capital letters.
7. Meta terms (such as 'flag') should rarely be part of the field name. 

Please fix those and we can merge it. Thanks!

Reply:
Thank you for the issues. They are fixed now. Thanks